Makefile

```diff
@@ -13,64 +13,69 @@ endif
 OPERATORCOURIER := $(shell command -v operator-courier 2> /dev/null)
 LICENSEFINDERBINARY := $(shell command -v license_finder 2> /dev/null)
 DEPENDENCY_DECISION_FILE = $(PROJECT_PATH)/doc/dependency_decisions.yml
+OPERATOR_SDK := operator-sdk
+GO := go
+OC := oc
+DOCKER := docker
 
 help: Makefile
 	@sed -n 's/^##//p' $<
 
 ## vendor: Populate vendor directory
 vendor:
```

Comment:
Vendoring is not required anymore, is it? 

This `vendor` target still useful?

Reply:
I mean, vendoring is required, obviously, but `vendor` directory at the root not required anymore, is it?

Reply:
Go modules do not exclude vendoring.
I see the vendor tag still useful to vendor the dependencies. operator-sdk command uses the vendor directory if its required in the project directory (https://github.com/operator-framework/operator-sdk/blob/master/CHANGELOG.md#removed-4)